app/src/main/java/com/quran/labs/androidquran/ui/SheikhAudioManagerActivity.kt

```diff
@@ -374,15 +374,19 @@ class SheikhAudioManagerActivity : QuranActionBarActivity(), SimpleDownloadListe
       holder.name.text = quranDisplayData.getSuraName(context, position + 1, true)
       val surahStatus: Int
       val surahStatusImage: Int
+      val surahStatusBackground: Int
```

Comment:
could we have just set the color directly as the background?

Reply:
We can but the button will be square. I wanted to stick to the current circular shape.
`holder.image.setBackgroundColor(Color.RED)` I think